ui/raidboss/data/07-dt/raid/r4s.ts

```diff
@@ -1,14 +1,1005 @@
+  },
+  combo: {
+    en: '${inOut} => ${bait}',
+  },
+  unknown: Outputs.unknown,
+} as const;
+
+const tailThrustOutputStrings = {
+  iceLeft: {
+    en: 'Double Knockback (<== Start on Left)',
+  },
+  iceRight: {
+    en: 'Double Knockback (Start on Right ==>)',
+  },
+  fireLeft: {
+    en: ' Start Front + Right ==>',
+  },
+  fireRight: {
+    en: '<== Start Front + Left',
+  },
+  unknown: Outputs.unknown,
+} as const;
+
+export interface Data extends RaidbossData {
+  phase: 1 | 2;
+  // Phase 1
+  bewitchingBurstSafe?: InOut;
+  hasForkedLightning: boolean;
+  seenBasicWitchHunt: boolean;
+  witchHuntBait?: NearFar;
+  witchHuntAoESafe?: InOut;
+  witchGleamCount: number;
+  electromines: { [id: string]: DirectionOutputIntercard };
+  electrominesSafe: DirectionOutputIntercard[];
+  starEffect?: PartnersSpread;
+  witchgleamSelfCount: number;
+  condenserTimer?: 'short' | 'long';
+  electronStreamSafe?: 'yellow' | 'blue';
+  electronStreamSide?: NorthSouth;
+  seenConductorDebuffs: boolean;
+  conductionPointTargets: string[];
+  // Phase 2
+  replicas: ActorPositions;
+  mustardBombTargets: string[];
+  aetherialEffect?: AetherialEffect;
+  twilightSafe: DirectionOutputIntercard[];
+  replicaCleaveCount: number;
+  secondTwilightCleaveSafe?: DirectionOutputIntercard;
+  midnightFirstMech?: PartnersSpread;
+}
 
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM4Savage',
   zoneId: ZoneId.AacLightHeavyweightM4Savage,
   timelineFile: 'r4s.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      phase: 1,
+      // Phase 1
+      hasForkedLightning: false,
+      seenBasicWitchHunt: false,
+      witchGleamCount: 0,
+      electromines: {},
+      electrominesSafe: [],
+      witchgleamSelfCount: 0,
+      seenConductorDebuffs: false,
+      conductionPointTargets: [],
+      // Phase 2
+      replicas: {},
+      mustardBombTargets: [],
+      twilightSafe: Directions.outputIntercardDir,
+      replicaCleaveCount: 0,
+    };
+  },
+  timelineTriggers: [
+    // Order: Soulshock => Impact x2 => Cannonbolt (entire sequence is ~9s).
+    // None of these have StartsUsing lines or other lines that could be used for pre-warn triggers;
+    // they seem to be entirely timeline based.  To avoid spam, use a single alert.
+    {
+      id: 'R4S Soulshock',
+      regex: /Soulshock/,
+      beforeSeconds: 4,
+      durationSeconds: 13,
+      response: Responses.bigAoe(),
+    },
+  ],
+  triggers: [
+    // ***************** PHASE 1 ***************** //
+    // General
+    {
+      id: 'R4S Phase Tracker',
+      type: 'Ability',
+      // 98D0 = Cannonbolt (knockback to south platform)
+      netRegex: { id: '98D0', source: 'Wicked Thunder', capture: false },
+      suppressSeconds: 1,
+      run: (data) => data.phase = 2,
+    },
+    {
+      id: 'R4S Wrath of Zeus',
+      type: 'StartsUsing',
+      netRegex: { id: '95EF', source: 'Wicked Thunder', capture: false },
+      response: Responses.aoe(),
```

Comment:
```suggestion
      response:  Responses.bigAoe(),
```

This kills without about 3~4 10%+ mits in week 1.